ndarray-linalg/Cargo.toml

```
@@ -37,7 +37,7 @@ rand = "0.8.3"
 thiserror = "1.0.24"
 
 [dependencies.ndarray]
-version = "0.15.2"
+version = "0.16.0"
```

```suggestion
version = ">=0.15.0, <0.17"
```

This would make it easier to use as a library downstream